src/config/env.js

```diff
@@ -30,6 +30,10 @@ export default Object.freeze({
     admin_password: process.env.ADMIN_PASSWORD,
     access_control_allow_origin: process.env.ACCESS_CONTROL_ALLOW_ORIGIN || "*",
 
+    // Deploy previews
+    is_heroku: process.env.IS_HEROKU,
```

Comment:
This variable is not needed, simply have an allowlist of allowed domains to query the api

Reply:
Also, the name would be misleading, this should not be coupled with any deployment service. What if we use vercel, netlify, digital ocean/ something else?

Reply:
All right then

Reply:
I believe this is a good implementation to what we want.

We will be using this regex in beta deploys (or heroku): `"https:\/\/deploy-preview-\d+--nijobs\.netlify\.app"`

Reply:
I don't think a regex is the way to go here. It solves the very specific problem we have now, sure, but what if there's a new provider we may want to use as well as netlify, what about local frontend builds? No one should have to know/understand this regex in order to allowlist something.

I think the ideal solution would be to make this variable a JSON string representing an array of regexes. It allows the solution you have now, but people can still add more domains easily, without having to think about regexes (exact strings are still valid regexes anyway)

In the end, the config object would parse the JSON into a JS array of Regexes, which can be evaluated against the requester's origin

Reply:
I think this might be the way to go. I left some examples in the readme and in the env file